test-suite-kotlin/src/test/kotlin/io/micronaut/kafka/docs/seek/rebalance/ProductClient.kt

```diff
@@ -8,6 +8,6 @@ import io.micronaut.kafka.docs.Product
 @Requires(property = "spec.name", value = "ConsumerRebalanceListenerTest")
 @KafkaClient
 interface ProductClient {
-    @Topic("awesome-products")
+    @Topic("fantastic-products")
```

Comment:
why are we changing from `awesome` to `fantastic` ?

Reply:
Because now there are other tests in the same project that produce records to this topic, and we probably don't want to start a fresh test container every time.